packages/nx-monorepo/src/nx-monorepo.ts

```diff
@@ -156,9 +197,34 @@ export class NxMonorepoProject extends TypeScriptProject {
     // Never publish a monorepo root package.
     this.package.addField("private", true);
 
-    // No need to compile or test a monorepo root package.
-    this.compileTask.reset();
-    this.testTask.reset();
+    // Add alias task for "projen" to synthesize workspace
+    this.addTask("synth-workspace", {
+      exec: "npx projen",
+      description: "Synthesize workspace",
+    });
+
+    this.addTask("run-many", {
+      receiveArgs: true,
```

Comment:
oo00oo is this property new?

Reply:
Yes it came out in Nov last year - great additional - https://projen.io/api/API.html#addtaskname-props-